base/admin-hpa.yaml

```diff
@@ -1,19 +1,19 @@
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
   name: admin-hpa
   namespace: notification-canada-ca
 spec:
-  minReplicas: 1
   maxReplicas: 2
-  scaleTargetRef:
-    apiVersion: apps/v1
-    kind: Deployment
-    name: admin
   metrics:
-  - type: Resource
-    resource:
+  - resource:
       name: cpu
       target:
-        type: Utilization
         averageUtilization: 50
+        type: Utilization
+    type: Resource
+  minReplicas: 1
```

Comment:
Nitpick: can we have the minReplicas and maxReplicas side by side, near each others?

Reply:
Yes good point - I ran this through a converter app and should have reorganized its output.